### DISCUSSION TITLE
如何通过 C++ 版键入非 ASCII 码内的字符？

### DISCUSSION THREAD

**Discussion by ghost** [yusancky/auto_type#14] [Q&A/疑问] [answered]
如题，如何通过 C++ 版的代码，键入非 ASCII 码内的字符，即键入 Unicode 字符？

**Comment by yusancky** [comment, marked as answer]
👋 Hi. 
I can understand your feeling.
But **C++ code can't type Unicode character** such as `雨` and `伞` is the biggest problem we meet. We recommend you to use Python code instead of C++ code. 
**We are finding solutions** now. Welcome to help us. 
Thanks for your support. Wish you can have a happy weekend.
